docs/03-Usage.md

```diff
@@ -2,7 +2,8 @@
 
 ## Getting started
 
-You first _need_ to create an index extending the `Firesphere\SolrSearch\Indexes\BaseIndex` class, or if you are using the compatibility
+The first thing you have to do is to create an index extending the
+`Firesphere\SolrSearch\Indexes\BaseIndex` class, or if you are using the compatibility
```

Comment:
I think the "need" part of the original should be included in some way or another. It's often mistaken as being optional, so a *need* to hard say it , might be useful

Reply:
Fair enough 👍